leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -383,7 +383,7 @@ var LinkRelationshipView = Backbone.View.extend({
         '<td><%= linkType %><%if (evolving){%> | <%}%> <%=postType %></td>',
         '<td class= "link-source"></td>',
         '<td class= "link-dest"></td>',
-        '<td><input class="linkAbsRelation" type="number" name="sth" value= <%- absTime %> > </td>',
+        '<td><input class="linkAbsRelation" type="number" name="sth" value= <%- absTP %> > </td>',
```

Comment:
Looks good! Wondering if we can remove the `name="sth"` as I don't think we really need it? What does it do

Reply:
I don't know what it does. I think maybe I should stick to the task? What do you think?

Reply:
Yeah it's fine! No need to change then